### ISSUE TITLE
tests fail with tox 2.0

### ISSUE THREAD

**Issue by mgedmin** [mgedmin/pyspacewar#3]
```
mg@platonas: ~/src/projects/pyspacewar [git:master=] $ tox
GLOB sdist-make: /home/mg/src/pyspacewar/setup.py
py27 inst-nodeps: /home/mg/src/pyspacewar/.tox/dist/pyspacewar-0.9.8dev.-git.104d24f-.zip
py27 installed: -f file:///home/mg/.cache/wheels,adium-theme-ubuntu==0.3.4,amqp==1.4.6,amqplib==1.0.2,ansible==1.7.2,anyjson==0.3.3,apsw==3.8.6.post1,apt-xapian-index==0.46,BeautifulSoup==3.2.1,beautifulsoup4==4.3.2,billiard==3.3.0.18,blinker==1.3,boto==2.34.0,bottle==0.12.8,bpython==0.12,Brlapi==0.6.2,bzr==2.7.0.dev1,bzr-builddeb==2.8.6,bzr-etckeeper==0.0.0,bzr-fastimport===0.13.0.final.0,BzrTools==2.6.0,CairoSVG==1.0.9,celery==3.1.13,Chameleon==2.16,characteristic==0.1.0,chardet==2.3.0,CherryPy==3.5.0,colorama==0.3.2,command-not-found==0.3,configglue==1.1.2,configobj==5.0.6,coverage==3.7.1,cssmin==0.2.0,cssselect==0.9.1,cssutils==0.9.10,Cython==0.21.1,debtagshw==0.1,decorator==3.4.0,defer==1.0.6,defusedxml==0.4.1,dirspec==13.10,distro-info==0.14,dnspython==1.12.0,docopt==0.6.2,docutils==0.12,driconf==0.9.1,duplicity==0.7.1,ecdsa==0.11,euca2ools==3.1.0,Fabric==1.10.0,fastimport==0.9.4,feedgenerator==1.7,feedparser==5.1.3,gbp==0.6.22,gevent==1.0.1,git-review==1.24,greenlet==0.4.5,gyp==0.1,html5lib==0.999,httpcode==0.5,httpie==0.8.0,httplib2==0.9,hypothesis==1.0.0,idna==0.9,iniparse==0.4,iotop==0.6,ipaddr==2.1.11,ipdb==0.8,ipython==2.3.0,Jinja2==2.7.3,key-mon==1.17,keyring==4.0,kombu==3.0.24,launchpadlib==1.10.3,lazr.restfulclient==0.13.4,lazr.uri==1.0.3,libvirt-python==1.2.12,lockfile==0.8,louis==2.5.3,lptools==0.2.0,lxml==3.4.2,M2Crypto==0.21.1,mailer==0.7,Mako==1.0.0,Markdown==2.6.1,MarkupSafe==0.23,matplotlib==1.4.2,mechanize==0.2.5,meld==3.12.3,meld3==1.0.0,meow==0.4.2,mercurial==3.1.2,mock==1.0.1,netifaces==0.10.4,nose==1.3.4,numpy==1.8.2,oauth==1.0.1,oauthlib==0.6.1,offlineimap==6.5.5,oneconf==0.3.7,PAM==0.4.2,paramiko==1.15.1,pdfshuffler==0.6.0,pelican==3.5.0,pep8==1.5.7,pexpect==3.2,Pillow==2.7.0,piston-mini-client==0.7.5,protobuf==2.6.1,py==1.4.25,pyasn1==0.1.7,pyasn1-modules==0.0.5,pyblosxom==1.5.3,pycrypto==2.6.1,pycups==1.9.72,pycurl==7.19.5,pydns==2.3.6,pyflakes==0.8.1,pygame===1.9.1release,pyglet==1.1.4,Pygments==2.0.1,pygobject==3.14.0,pygpgme==0.3,pygraphviz==1.3rc2,pyinotify==0.9.5,pyOpenSSL==0.13.1,pyparsing==2.0.3,pyPdf==1.13,PyPDF2==1.23,Pyrex==0.9.8.5,pyserial==2.6,pysmbc==1.0.15.3,pyspacewar===0.9.8dev.-git.-,pysqlite==1.0.1,pytest==2.6.3,python-apt===0.9.3.11build1,python-cloudfiles==1.7.10,python-dateutil==2.2,python-debian==0.1.22,python-debianbts==1.11,python-distutils-extra==2.38,python-memcached==1.53,python-xlib==0.14,pytz==2014.10,pyxdg==0.25,PyYAML==3.11,qrtools==1.4,reportbug===6.6.3ubuntu1,reportlab==3.1.44,repoze.lru==0.6,requestbuilder==0.2.3,requests==2.4.3,roman==2.0.0,Routes==2.0,screenkey==0.2,SecretStorage==2.1.1,service-identity==1.0.0,sessioninstaller==0.0.0,simplegeneric==0.8.1,simplejson==3.6.5,six==1.9.0,SOAPpy==0.12.22,software-center-aptd-plugins==0.0.0,Sphinx==1.2.3,sphinxcontrib-issuetracker==0.11,supervisor==3.0,system-service==0.1.6,Twisted-Core==14.0.2,Twisted-Web==14.0.2,ubuntu-dev-tools===0.155-ubuntu2,Unidecode==0.4.16,unity-lens-photos==1.0,urlgrabber==3.9.1,urllib3==1.9.1,uTidylib==0.2,vboxapi==1.0,virtkey==0.63.0,virtualenv==1.11.6,wadllib==1.3.2,WebOb==1.4,wheel==0.24.0,winpdb==1.4.8,wstools==0.4.3,wxPython==3.0.1.1,wxPython-common==2.8.12.1,xdiagnose==3.7.2,xdot==0.6,xkit==0.0.0,xlwt==0.7.5,xmldiff==0.6.10,youtube-dl==2015.2.28,zenmap==6.47,zope.interface==4.1.1
py27 runtests: PYTHONHASHSEED='1950202100'
py27 runtests: commands[0] | python test.py
....................................Error opening terminal: unknown.
ERROR: InvocationError: '/home/mg/src/pyspacewar/.tox/py27/bin/python test.py'
______________________________________________________________________ summary _______________________________________________________________________
ERROR:   py27: commands failed

[exited with 1]
```

**Comment by mgedmin** [comment]
I can reproduce with `TERM= DISPLAY= python test.py`.  Leaving just one of the two unset makes the tests pass (although without DISPLAY there's a very weird screen flash).

**Comment by mgedmin** [comment]
My [half-baked test runner](https://github.com/mgedmin/pyspacewar/blob/master/test.py) causes pain by not printing test names.  Time to switch to something else (nose?).

**Comment by mgedmin** [comment]
This was fixed in 61cd017fb7bf60c5e53a0abdd4349fdca5582790.
